frontend/src/components/widgets/Form/Form.tsx

```diff
@@ -33,11 +33,20 @@ interface State {
   showMissingSubmitButtonWarning: boolean
 }
 
+/**
+ * If the form doesn't have a submit button after this many milliseconds
+ * have elapsed, we show a warning that tells them they probably want to
+ * create one.
+ */
+export const SUBMIT_BUTTON_ERROR_TIME_MS = 1500
+
```

Comment:
This looks like it snuck in from a bad rebase, maybe? The timer is no longer part of the form implementation, so this should be removed.